fluent-react/test/localized_fallback.test.js

```diff
@@ -61,6 +61,8 @@ foo = FOO
 });
 
 test("falls back back for missing message", function() {
+  jest.spyOn(console, "warn").mockImplementation(() => {});
```

Comment:
This activates Jests's "spy" feature, which allows the test to see if the function was called. In general I prefer this approach as it tests the "user interface" for the code.